marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/flow-utils/getMetadata.sjs

```diff
@@ -3,43 +3,25 @@ const test = require("/test/test-helper.xqy");
 const FlowUtils = require("/data-hub/5/impl/flow-utils.sjs");
 const flowUtils = new FlowUtils();
 
-const results = [];
-const metadata = [];
-
-let jobId = "alpha-num3r1c-j06-1d";
-let flowDoc = {
-                "name": "testFlow",
-                "description": "This is the flow doc to test metadata values",
-                "steps": {
-                  "1": {
-                    "name": "ingestion",
-                    "description": "This is the custom ingestion step",
-                    "stepDefinitionName": "ingestion",
-                    "stepDefinitionType": "INGESTION"
-                  },
-                  "2": {
-                    "name": "mapping",
-                    "description": "This is the custom mapping step",
-                    "stepDefinitionName": "mapping",
-                    "stepDefinitionType": "MAPPING"
-                  },
-                  "3": {
-                     "name": "mastering",
-                     "description": "This is the custom mapping step",
-                     "stepDefinitionName": "mastering",
-                     "stepDefinitionType": "MASTERING"
-                  }
-                }
-              };
-
-//this is to verify that stepDefName corresponding to the stepNumber is the value of metadata datahubCreatedByStep
-for (const stepNum in flowDoc.steps) {
-  metadata.push(flowUtils.createMetadata(metadata ? metadata : {}, flowDoc.name, flowDoc.steps[stepNum].stepDefinitionName, jobId).datahubCreatedByStep)
```

Comment:
This test wasn't really testing the function, because it was passing in values - one of which was incorrect (the stepDefinitionName). I reworked the test to instead verify the interesting logic, which is that each bit is overwritten except for jobId.